x/evm/keeper/grpc_query.go

```diff
@@ -485,100 +485,75 @@ func (k *Keeper) traceTx(
 ) (*interface{}, uint, error) {
 	// Assemble the structured logger or the JavaScript tracer
 	var (
-		tracer    vm.EVMLogger
+		tracer    tracers.Tracer
 		overrides *ethparams.ChainConfig
 		err       error
+		timeout   = defaultTraceTimeout
 	)
-
 	msg, err := tx.AsMessage(signer, cfg.BaseFee)
 	if err != nil {
 		return nil, 0, status.Error(codes.Internal, err.Error())
 	}
 
-	if traceConfig != nil && traceConfig.Overrides != nil {
+	if traceConfig == nil {
+		traceConfig = &types.TraceConfig{Debug: true}
```

Comment:
why is `Debug` true?

Reply:
That is to stay consistant with our previous version. In  https://github.com/evmos/ethermint/blob/6a0b8a38d25b207259dc989848377336a56b22d1/x/evm/types/tracer.go#L28. 

Although I will remove it as this differs from the Geth implementation